docs/adding_packages/README.md

```diff
@@ -3,180 +3,131 @@
 ConanCenterIndex aims to provide the best quality packages of any open source project.
 Any C/C++ project can be made available by contributing a "recipe".
 
-Getting started is easy. Try building an existing package with our [developing recipes](../developing_recipes_locally.md) tutorial.
-To deepen you understanding, start with the [How to provide a good recipe](#how-to-provide-a-good-recipe) section.
+Getting started is easy. Try building an existing package with our [developing recipes](../developing_recipes_locally.md)
+tutorial. To deepen you understanding, start with the [How to provide a good recipe](#how-to-provide-a-good-recipe) section.
 You can follow the three steps (:one: :two: :three:) described below! :tada:
 
 <!-- toc -->
 ## Contents
 
-  * [Request access](#request-access)
+  * [:one: Request access](#one-request-access)
   * [Inactivity and user removal](#inactivity-and-user-removal)
-  * [Submitting a Package](#submitting-a-package)
-    * [The Build Service](#the-build-service)
-  * [Recipe files structure](#recipe-files-structure)
-    * [`config.yml`](#configyml)
-    * [`conandata.yml`](#conandatayml)
-    * [The _recipe folder_: `conanfile.py`](#the-_recipe-folder_-conanfilepy)
-    * [Test Folders](#test-folders)
-  * [Test the recipe locally](#test-the-recipe-locally)
-    * [Hooks](#hooks)
-    * [Linters](#linters)<!-- endToc -->
-
-## Request access
-
-:one: The first step to add packages to ConanCenter is requesting access. To enroll in ConanCenter repository, please write a comment
-requesting access in this GitHub [issue](https://github.com/conan-io/conan-center-index/issues/4). Feel free to introduce yourself and
+  * [:two: Creating a package](#two-creating-a-package)
+    * [How to provide a good recipe](#how-to-provide-a-good-recipe)
+  * [:three: Submitting a Package](#three-submitting-a-package)
+  * [The Build Service](#the-build-service)<!-- endToc -->
+
+## :one: Request access
+
+The first step to add packages to ConanCenter is requesting access. To enroll in ConanCenterIndex repository, please write a comment
+requesting access in this GitHub [issue #4](https://github.com/conan-io/conan-center-index/issues/4). Feel free to introduce yourself and
 your motivation to join ConanCenter community.
 
-This process helps ConanCenter against spam and malicious code. The process is not not automated on purpose and the requests are generally approved
-on a weekly basis.
+This process helps ConanCenter against spam and malicious code. The process is not fully automated on purpose and the requests are
+generally approved on a weekly basis. Feel free to continue to step :two: while waiting for approval.
```

Comment:
twice a week, I suppose

Reply:
Let's under promise and over deliver, theres sometimes delays in them rolling out